website/web/genericapi.py

```diff
@@ -380,7 +380,19 @@ class CaptureCookies(Resource):  # type: ignore[misc]
     def get(self, capture_uuid: str) -> dict[str, Any]:
         return json.loads(lookyloo.get_cookies(capture_uuid).read())
 
-
+@api.route('/json/<string:capture_uuid>/report')
+@api.doc(description='Reports the url by sending an email to the investigation team',
+         params={'capture_uuid': 'The UUID of the capture'})
+class CaptureReport(Resource):  # type: ignore[misc]
+    @api.param('email', 'Email of the reporter, used by the analyst to get in touch.')  # type: ignore[misc]
+    @api.param('comment', 'Description of the URL, will be given to the analyst.')  # type: ignore[misc]
+    def post(self, capture_uuid) -> str:
+        parameters: dict[str, Any] = request.get_json(force=True)
+        answer = lookyloo.send_mail(capture_uuid, parameters.get('email'), parameters.get('comment'))
```

Comment:
This method (`lookyloo.send_mail`) needs to be updated: it always returns `None` regardless if it worked or not (and throws a log entry if it doesn't). It wasn't implemented with the API in mind.

It should return `True` if it worked, and a dict if it didn't (`{"error": "<error message>"}`). Can you update `lookyloo.send_mail` so it does that instead? The error message should be `Unable to send mail.` in the last exception bloc. 
And if there are no exception, `return True` at the very end of the method.

You will also need to update the typing.

When it's done, you can directly do `return lookyloo.send_mail(...` because the method will return what we want to send back to the user.